### DISCUSSION TITLE
[Projects Beta] I can assign an iteration to an issue, but I can not remove it

### DISCUSSION THREAD

**Discussion by tcbrouwer** [community/community#11894] [Projects and Issues]
Issues are created without an assigned iteration. I can assign an iteration to an issue. If I want to change it I can get a dropdown to set it to a different iteration. However there is no option to reset the value to it's original [none].

It feels like there should be a way to do this.

My current workaround is that I have created a dummy iteration. If I want to set an issue's iteration to [none], I set it to this dummy value instead.

![image](https://user-images.githubusercontent.com/17088336/155094058-61bf6306-4b78-482e-a677-6d3f49141ba6.png)

btw the same seems to be true for the 'Single select' field type.

**Comment by aikenka** [comment]
If you select the cell, and press the **delete** key on your keyboard, it will clear the value.

1. select cell:  <br/> ![image](https://github.com/community/community/assets/64041856/de18ec78-c6cd-43b7-9d5b-2bf617018aee)
4. press **delete** key on your keyboard
5. cell is now empty <br/> ![image](https://github.com/community/community/assets/64041856/9138dc80-e95f-441f-b07c-d6dce78c43e4)

**Reply by wbudd** [reply]
There is no **delete** (button? icon?) to click. And I think you know that, because otherwise you would have added a screenshot for step 2 too, as you so helpfully did for step 1 and 3.

**Reply by aikenka** [reply]
You are correct there is no **delete** button on the screen. I should have said click the **delete key** on your keyboard. I'll edit the steps above to be more clear.

**Reply by wbudd** [reply]
Ohhhhhh, that does actually work, doesn't it! As you say, you need to click the cell first to give it focus for the delete key press to register. I missed that when I tried the del key earlier, because the focus was on the dropdown menu instead.

This seems more like an Easter Egg than any kind of UI design, so kudos for finding that! (But maybe in the future, avoid the verb "click" for pressing stuff on the keyboard 💀)

**Reply by aikenka** [reply]
Good call - I updated "click" to "press".

Big time Easter Egg! I credit any abilities this way to too much time playing Mario Kart.
